packages/click-to-react-component/src/ClickToComponent.js

```diff
@@ -198,12 +198,11 @@ export function ClickToComponent({ editor = 'vscode' }) {
 
       [data-click-to-component-target] {
         cursor: var(--click-to-component-cursor, context-menu) !important;
+        outline: auto 1px;
         outline: var(
           --click-to-component-outline,
-          2px solid lightgreen
+          -webkit-focus-ring-color auto 1px
         ) !important;
-        outline-offset: -2px;
-        outline-style: inset;
```

Comment:
Have you checked to see if this is still visible on the root `<App />` in CRA?

When I was testing things initially, you couldn't tell the component was highlighted because the outline is _outside_ of the component (and outside of the viewport!).

This offset moved it to the inside so anything that extends to the edges of the viewport still got the outline.

Reply:
Just tested this out & the behavior seems fine!

TIL!

It looks like the browser automatically handles light/dark contrasts, too!

Reply:
Yeah~ That's the magic that browser handles for us.